### PR TITLE
Adding fuse-ii borrowing widget

### PR DESCRIPTION
**Pull request by IanWoodard**

### CODE REVIEW

File: earn/src/components/lend/BorrowingWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+      return borrowEntries;
+    }
+    return Object.entries(borrowEntries).reduce((filtered, [key, entries]) => {
+      const filteredEntries = entries.filter((entry) => entry.collateral.symbol === selectedCollateral.asset.symbol);
+      if (filteredEntries.length > 0) {
+        filtered[key] = filteredEntries;
+      }
+      return filtered;
+    }, {} as { [key: string]: BorrowEntry[] });
+  }, [borrowEntries, selectedCollateral]);
```
Comment (haydenshively) [inline earn/src/components/lend/BorrowingWidget.tsx:141]: Comments. Very confused

File: earn/src/components/lend/BorrowingWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+                              {account.token0.symbol}
+                            </Display>
+                          </div>
+                        </AvailableContainerConnectedRight>
+                      )}
+                      {hasAssetsForToken0 && !hasLiabilitiesForToken1 && <AvailableContainerPlaceholder />}
+                      {hasLiabilitiesForToken1 && (
+                        <AvailableContainerConnectedRight $backgroundGradient={token1Gradient}>
+                          <Display size='XXS' color={SECONDARY_COLOR}>
+                            3% APY
```
Comment (haydenshively) [inline earn/src/components/lend/BorrowingWidget.tsx:319]: ?

File: earn/src/components/lend/BorrowingWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+                  const token1Color = tokenColors.get(account.token1.address);
+                  const token1Gradient = token1Color
+                    ? `linear-gradient(90deg, ${GREY_700}  0%, ${rgba(token1Color, 0.25)} 100%)`
+                    : undefined;
+                  return (
+                    <Fragment key={account.uniswapPool}>
+                      {hasLiabilitiesForToken0 && (
+                        <AvailableContainerConnectedRight $backgroundGradient={token0Gradient}>
+                          <Display size='XXS' color={SECONDARY_COLOR}>
+                            3% APY
```
Comment (haydenshively) [inline earn/src/components/lend/BorrowingWidget.tsx:303]: ?

**Review by haydenshively — APPROVED** [review]

File: earn/src/components/lend/BorrowingWidget.tsx (hunk 2 above)
Comment (IanWoodard) [reply earn/src/components/lend/BorrowingWidget.tsx:319]: Didn't get around to computing the actual value here

File: earn/src/components/lend/BorrowingWidget.tsx (hunk 3 above)
Comment (IanWoodard) [reply earn/src/components/lend/BorrowingWidget.tsx:303]: Didn't get around to computing the actual value here

**Pull request merged**